### DISCUSSION TITLE
System architecture

### DISCUSSION THREAD

**Discussion by TemirlanAidarov** [webdriverio/webdriverio#8222] [General]
Hello!
Is there any documents that describes the system architecture of WebDriverIO?

**Comment by erwinheitzman** [comment]
There is but is seems that there's some issues with the flowcharts of our architecture showing up on the website. I have shared the issue with the rest of the team to see if anyone knows what might be the issue so that we can resolve this.

For now I'll share some links to the old website where the charts still work: 
https://v5.webdriver.io/docs/flowcharts/wdiocommands.html
https://v5.webdriver.io/docs/flowcharts/createlocalworkerprocess.html
https://v5.webdriver.io/docs/flowcharts/testexecution.html
https://v5.webdriver.io/docs/flowcharts/highleveloverview.html

**Comment by christian-bromann** [comment]
Would love to bring these diagrams back to the docs page.
